src/alchemlyb/parsing/amber.py

```diff
@@ -166,16 +169,12 @@ def __init__(self):
 def file_validation(outfile):
     """validate the energy output file """
```

Comment:
Could you please add some additional information here. I know that you didn't write this code but you have dealt with it now for a long time so that _you_ really know well what _should_ be written here. This is especially important if we then want to change this function later.

I'd like to see a summary of what the function does, definition of input and return values, and conditions that raise exceptions.

Reply:
Sure, I'll edit the docstring now, what do you think about the suggestion of changing the method name in issue #238 ?
`file_validation` is a little reductive, as this function not only "validates" the file, but also reads it and creates a corresponding FEData object. What do you think?